android/titanium/src/java/org/appcelerator/titanium/TiBaseActivity.java

```diff
@@ -763,17 +763,18 @@ public void onChanged(TiActivitySafeAreaMonitor monitor)
 			}
 
 			@Override
-			public void onKeyboardChanged(boolean keyboardVisible)
+			public void onKeyboardChanged(boolean isVisible)
 			{
-				if (keyboardVisible != oldKeyboardValue) {
-					KrollDict kd = new KrollDict();
-					kd.put("keyboardVisible", keyboardVisible);
+				if (isVisible != keyboardVisible) {
 					TiWindowProxy windowProxy = TiBaseActivity.this.window;
 					if (windowProxy != null) {
 						windowProxy.fireSafeAreaChangedEvent();
 					}
-					windowProxy.fireEvent("keyboardChanged", kd);
-					oldKeyboardValue = keyboardVisible;
+					KrollDict kd = new KrollDict();
+					kd.put("keyboardFrame", null);
```

Comment:
Could we eventually use this snippet to get some parts of the frame? https://stackoverflow.com/a/72566595/5537752

Reply:
added those values but I'll need to verify it with some different keyboards. height/width is correct on my Samsung A20.